recipes/asyncify/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "asyncify" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a3/c6/33d42a1920395c9cfe912d4b6425504237b6bba47a48967aaf453b2b0d79/asyncify-0.2.2.tar.gz
```

Could you please change it to:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```